### PR TITLE
Brenner/devtools

### PR DESCRIPTION
**Pull request by BrennerSpear**
Makes importing df-related utils into your console as easy as clicking a button.
Also added Artifactory utils to the utils file

### CODE REVIEW

File: javascript/utils/utils.js [hunk 1]
```diff
@@ -69,3 +69,46 @@ export let canHaveArtifact = (planet) => {
...
+
+export let calcBonus = (bonus) => {
+  return bonus - 100
+}
+
+export let myPlanetsWithArtifacts = () => {
+  return Array.from(df.getMyPlanets())
+    .filter(df.isPlanetMineable)
+    .sort((p1, p2) => parseInt(p1.locationId, 16) - parseInt(p2.locationId, 16));
+}
```
Comment (phated) [inline javascript/utils/utils.js:85]: This naming is actually false advertising and was causing a bug in artifactory. Can you snag the new methods/names from the latest version I made available? I think the name of this became `myPlanetsWithFindable` and I added the correct implementation for `myPlanetsWithArtifacts`

File: javascript/utils/utils.js [hunk 2]
```diff
@@ -69,3 +69,46 @@ export let canHaveArtifact = (planet) => {
 
 export let canFindArtifact = (planet) => energy(planet) >= 96;
 export let hasArtifact = (planet) => planet.heldArtifactId != null;
+
+export let canDeposit = (planet) => {
+  return planet && isMine(planet) && !planet.heldArtifactId
+}
+
+export let calcBonus = (bonus) => {
```
Comment (phated) [inline javascript/utils/utils.js:77]: let's rename to
```suggestion
export let calcArtifactBonus = (bonus) => {
```

File: javascript/utils/utils.js [hunk 3]
```diff
@@ -69,3 +69,46 @@ export let canHaveArtifact = (planet) => {
 
 export let canFindArtifact = (planet) => energy(planet) >= 96;
 export let hasArtifact = (planet) => planet.heldArtifactId != null;
+
+export let canDeposit = (planet) => {
```
Comment (phated) [inline javascript/utils/utils.js:73]: let's rename to
```suggestion
export let canDepositArtifact = (planet) => {
```

**Review by phated — CHANGES_REQUESTED** [review]
Very cool. I love how this will improve the prototype experience. I didn't account for UI-less plugins in the showcase, so maybe put the console.log message into the UI and take a screenshot?

**Comment by jacobrosenthal** [comment]
Always happy to have it reopened

**Pull request closed without merging**